README.md

```diff
@@ -59,32 +59,19 @@ The application has jasmine tests, which can be accessed at `/specs` when the ap
 2. Publish a Finder Content Item to the content store. See the doc for [Finder Content Item](https://github.com/alphagov/finder-frontend/blob/master/docs/finder-content-item.md) for more info.
 3. Ensure your documents are indexed in [alphagov/rummager](http://github.com/alphagov/rummager) correctly.
 
+### How can I know the filter needed?
```

Comment:
this title still sounds a bit strange to me - how about calling it something like "How to add a fixed filter"?